<launch>
   <node pkg="final" type="line_follower.py" name="line_follower" output = "screen">

   <param name="plan_topic" type="string" value="/planner_node/car_plan"  />
   <param name="pose_topic" type="string" value="pf/viz/inferred_pose" />
   <param name="plan_lookahead" type="int" value="5" />
   <param name="translation_weight" type= "double" value="1.0" />
   <param name="rotation_weight" type= "double" value="0.39" />
   <param name="kp" type= "double" value="5.0" />
   <param name="ki" type= "double" value="0.7" />
   <param name="kd" type= "double" value="1.5" />
   <param name="error_buff_length" type= "int" value="10" />
   <param name="speed" type= "double" value="1.0" />

  </node>


</launch>
